packages/styles/src/variables/components/_form-check.scss

```
@@ -14,7 +14,28 @@ $form-check-input-size: 1.375rem !default;
 $form-check-input-size-sm: spacing.$size-regular !default;
 $form-check-input-border-width: forms.$input-border-width !default;
 $form-check-input-focus-box-shadow: forms.$input-focus-box-shadow !default;
+$form-check-input-disabled-svg-border-width: $form-check-input-border-width * 2;
+$form-check-input-disabled-background-url-light: "data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' stroke='" +
```

For me it's unclear why there are numbers behind the percentage character. And as far as I can see, they don't do anything.
```suggestion
$form-check-input-disabled-background-url-light: "data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%' height='100%' fill='none' stroke='" +
```

---

It's percent encoding: https://en.wikipedia.org/wiki/Percent-encoding#Reserved_characters
Looks fine to me.